libs/ui/layout/src/lib/download-entry/download-entry.component.stories.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  moduleMetadata,
+  Story,
+  Meta,
+  componentWrapperDecorator,
+} from '@storybook/angular'
+import { DownloadEntryComponent } from './download-entry.component'
+import { UiLayoutModule } from '../ui-layout.module'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
+
+export default {
+  title: 'Layout/DownloadEntryComponent',
```

Comment:
I would start a section `Dataset`, to me putting everything in the `layout` one will be a mess in the end.

Reply:
Ok, thanks, do I understand correctly that you would create a `ui-dataset.module`?

Reply:
I would at least create a new section in the storybook tree.
```js
title: 'Dataset/DownloadEntryComponent
```
but actually i think it would be good to refactor a bit the libraries yes, the `layout` is a bit too general, also we need to think about what we do put in `feature-search`, is it a library for the search in itself (state, services, eventually smart component), or the logic for the search UI ?
I would have at least an Angular module for `mdView` and `dataset` which would contain the smart components.
@jahow any opinion ?

Reply:
Yes, maybe we should clarify and agree on how smart and presentation modules are categorized.

I'll create a documentation PR with a proposition and we can keep talking about it there.

For now though, I think the `ui-layout` module is fine for these components.

Reply:
doc PR here https://github.com/geonetwork/geonetwork-ui/pull/154

Reply:
The discussed refactoring has been in merged in #157